src/reflect/scala/reflect/internal/SymbolTable.scala

```diff
@@ -467,11 +467,13 @@ abstract class SymbolTable extends macros.Universe
   }
 
   /** The set of all installed infotransformers. */
-  var infoTransformers = new InfoTransformer {
+  val _infoTransformers: ThreadLocalStorage[InfoTransformer] = ThreadLocalStorage(new InfoTransformer {
```

Comment:
that is not a threadlocal - it relates to the operations of a phase

it should not be modified during a phase, maybe not during a run